src/cgame/cg_rocket_draw.cpp

```diff
@@ -917,6 +917,31 @@ class SpeedGraphElement : public HudElement
 	Color::Color backColor;
 };
 
+class PositionElement : public TextHudElement
+{
+public:
+	PositionElement( const Rml::String& tag ) :
+			TextHudElement( tag, ELEMENT_GAME, true )
+	{
+	}
+
+	void DoOnRender() override
+	{
+		if ( !cg_drawPosition.Get() )
+		{
+			SetText( "" );
+			return;
+		}
+
+		if ( cg.predictedPlayerState.clientNum == cg.clientNum )
```

Comment:
tbh I don't see why we should hide this (and the speedometer) when following someone else, for example a bot. I would just remove the condition and activate it always, but shrug.

Reply:
Hmm, I also see no reason to hide something when spectating.